sycl/test-e2e/WeakObject/weak_object_copy.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
-// RUN: %BE_RUN_PLACEHOLDER %t.out
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
```

Comment:
I'm not sure why we used `BE_RUN_PLACEHOLDER` instead of a usual triple of placeholders, but I think the "triple" behavior is the correct one for those tests.

Reply:
I think it was done to avoid running the tests multiple times. These tests check the weak_object wrapper and don't execute device code, so there's not much value in running them for each device type.

Reply:
Fair enough, I don't have strong objections to that.